cmd/root_test.go

```diff
@@ -1,10 +1,23 @@
 package cmd
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/go-logr/logr"
+)
+
+type mockLogger struct{}
+
+func (ml mockLogger) Error(err error, msg string, keysAndValues ...interface{}) {}
+func (ml mockLogger) Info(msg string, keysAndValues ...interface{})             {}
+func (ml mockLogger) Enabled() bool                                             { return false }
+func (ml mockLogger) WithName(name string) logr.Logger                          { return mockLogger{} }
+func (ml mockLogger) WithValues(keysAndValues ...interface{}) logr.Logger       { return mockLogger{} }
+func (ml mockLogger) V(level int) logr.Logger                                   { return mockLogger{} }
 
 func TestRoot(t *testing.T) {
 	t.Run("Execute", func(t *testing.T) {
-		if err := Execute(); err != nil {
+		if err := Execute(mockLogger{}); err != nil {
```

Comment:
I think my phrasing around "mock" logger likely wasn't clear so that's my bad - what I had in mind was passing the [`NewTestingLogger(t)`](https://pkg.go.dev/github.com/go-logr/logr/testing#NewTestLogger) function.

Reply:
Fixed this up, my bad. Should have asked for clarification.